packages/instant/src/components/payment_method_dropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import { BigNumber } from '@0x/utils';
+import copy from 'copy-to-clipboard';
+import * as React from 'react';
+
+import { Network } from '../types';
+import { etherscanUtil } from '../util/etherscan';
+import { format } from '../util/format';
+
+import { Dropdown, DropdownItemConfig } from './ui/dropdown';
+
+export interface PaymentMethodDropdownProps {
+    selectedEthAddress: string;
```

Comment:
lets call these `accountAddress` and `accountEthBalanceInWei` to remain consistent with the address / balance fetching branch